### ISSUE TITLE
f5_ve_standalone_2_nic refers to broken GitHub URL (liberty branch)

### ISSUE THREAD

**Issue by andrewjjenkins** [F5Networks/f5-openstack-heat#89]
f5_ve_standalone_2_nic includes resources like bigip_data_security_group and bigip_mgmt_security_group that have URLs to github that are no longer valid (they point at the develop branch).

```
  bigip_data_security_group:
    type: https://raw.githubusercontent.com/F5Networks/f5-openstack-heat/develop/unsupported/ve/common/bigip_data_security_group.yaml
  bigip_mgmt_security_group:
    type: https://raw.githubusercontent.com/F5Networks/f5-openstack-heat/develop/unsupported/ve/common/bigip_mgmt_security_group.yaml
```

**Comment by pjbreaux** [comment]
Fixing now.

**Comment by pjbreaux** [comment]
Fixed by PR #88
